lib/elixir/lib/calendar.ex

```diff
@@ -194,6 +234,16 @@ defmodule Date do
   end
 
   @doc """
+  Returns the number of days in a week.
+
+  Always returns 7 days in a week irrespective of the
+  calendar in use.
+  """
+  def days_in_a_week() do
```

Comment:
José, this is a bit lazy of me on two fronts:

1.  All of the calendars in D&R that use a format `{year, month, day}` are 7 days in a week. 
2. For calculations on `{RD, milliseconds}` (or my original case of `integer`} like the `kday` functions, there is no longer the concept of a calendar so an assumption has to be made.

I suspect you may consider the `kday` functions too much for the expressed intent of the core Calendar module and therefore I can certainly extract them.  But they, and the Enum protocol support, are extremely useful for anyone doing calendar work (calculating public holidays / festivals / religious and cultural events) which I why I proposed including them.